packages/envconfig/src/envconfig-toml.ts

```diff
@@ -0,0 +1,326 @@
+import { readFileSync } from 'fs';
+import * as path from 'path';
+import * as os from 'os';
+import { parse, stringify, TomlTable } from 'smol-toml';
+import {
+  ConfigDataSource,
+  LoadClientConfigOptions,
+  LoadClientProfileOptions,
+  TomlClientConfig,
+  TomlClientConfigCodec,
+  TomlClientConfigProfile,
+  TomlClientConfigTLS,
+} from './types';
+import { normalizeGrpcMetaKey } from './utils';
+
+function sourceToStringData(source?: ConfigDataSource): string | undefined {
+  if (source === undefined) {
+    return undefined;
+  }
+  if ('path' in source) {
+    return readFileSync(source.path)?.toString();
```

Comment:
Pass `{ encoding: 'utf-8' }` as the second argument to `readFileSync`, and its return value will be a string. No need for the extra `toString()` call.